### PR TITLE
Add hcp packer run task

### PR DESCRIPTION
**Pull request by laurapacilio**
This PR adds a section to the [Terraform Run Tasks Integration page](https://www.terraform.io/cloud-docs/integrations/run-tasks) about the new HCP Packer run task.

### CODE REVIEW

**Comment by laurapacilio** [comment]
@im2nguyen I would love to link to the learn tutorials here :-) Once you've got the titles and expected URLs would you mind dropping them in as a suggestion? :-)

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 1]
```diff
@@ -78,6 +78,14 @@ The signature is a sha512 sum of the webhook body using the provided HMAC key. T
 $ echo -n $WEBHOOK_BODY | openssl dgst -sha512 -hmac "$HMAC_KEY"
 ```
 
+## HCP Packer Run Task
+
```
Comment (im2nguyen) [inline content/cloud-docs/integrations/run-tasks/index.mdx:82]:
```suggestion
-> **Hands On:** Try the [Set Up Terraform Cloud Run Task for HCP Packer](https://learn.hashicorp.com/tutorials/packer/setup-tfc-run-task), [Standard tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-data-source-image-validation), and [Plus tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-resource-image-validation) tutorials on HashiCorp Learn to set up and test the Terraform Cloud Run Task integration end to end.
```

**Review by im2nguyen — APPROVED** [review]

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 2]
```diff
@@ -78,6 +78,14 @@ The signature is a sha512 sum of the webhook body using the provided HMAC key. T
 $ echo -n $WEBHOOK_BODY | openssl dgst -sha512 -hmac "$HMAC_KEY"
 ```
 
+## HCP Packer Run Task
+-> **Hands On:** Try the [Set Up Terraform Cloud Run Task for HCP Packer](https://learn.hashicorp.com/tutorials/packer/setup-tfc-run-task), [Standard tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-data-source-image-validation), and [Plus tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-resource-image-validation) tutorials on HashiCorp Learn to set up and test the Terraform Cloud Run Task integration end to end.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:82]:
```suggestion
> **Hands On:** Try the [Set Up Terraform Cloud Run Task for HCP Packer](https://learn.hashicorp.com/tutorials/packer/setup-tfc-run-task), [Standard tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-data-source-image-validation), and [Plus tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-resource-image-validation) tutorials on HashiCorp Learn to set up and test the Terraform Cloud Run Task integration end to end.
```

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 3]
```diff
@@ -78,6 +78,14 @@ The signature is a sha512 sum of the webhook body using the provided HMAC key. T
 $ echo -n $WEBHOOK_BODY | openssl dgst -sha512 -hmac "$HMAC_KEY"
 ```
 
+## HCP Packer Run Task
+> **Hands On:** Try the [Set Up Terraform Cloud Run Task for HCP Packer](https://learn.hashicorp.com/tutorials/packer/setup-tfc-run-task), [Standard tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-data-source-image-validation), and [Plus tier run task image validation](https://learn.hashicorp.com/tutorials/packer/run-tasks-resource-image-validation) tutorials on HashiCorp Learn to set up and test the Terraform Cloud Run Task integration end to end.
+[Packer](https://www.packer.io/) lets you create identical machine images for multiple platforms from a single source template. The [HCP Packer registry](docs/packer) lets you track golden images, designate images for test and production environments, and query images to use in Packer and Terraform configurations.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:83]:
```suggestion

[Packer](https://www.packer.io/) lets you create identical machine images for multiple platforms from a single source template. The [HCP Packer registry](docs/packer) lets you track golden images, designate images for test and production environments, and query images to use in Packer and Terraform configurations.
```

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 4]
```diff
@@ -78,6 +78,14 @@ The signature is a sha512 sum of the webhook body using the provided HMAC key. T
...
+
+The HCP Packer validation run task checks the image artifacts within a Terraform configuration. If the configuration references images marked as unusable (revoked), the run task fails and users receive an error message containing the number of revoked artifacts and whether HCP Packer has metadata for newer versions. For HCP Packer Plus registries, run tasks also help you identify hardcoded and untracked images that may not meet security and compliance requirements.
+
+To get started, [create an HCP Packer account](/products/packer) and follow the instructions in the [HCP Packer Run Task](/docs/packer/reference-image-metadata#terraform-cloud-run-tasks) documentation.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:87]:
```suggestion
To get started, [create an HCP Packer account](/products/packer) and follow the instructions in the [HCP Packer Run Task](/docs/packer/manage-image-use/terraform-cloud-run-tasks) documentation.
```

File: content/cloud-docs/integrations/run-tasks/index.mdx [hunk 5]
```diff
@@ -78,6 +78,14 @@ The signature is a sha512 sum of the webhook body using the provided HMAC key. T
...
+[Packer](https://www.packer.io/) lets you create identical machine images for multiple platforms from a single source template. The [HCP Packer registry](docs/packer) lets you track golden images, designate images for test and production environments, and query images to use in Packer and Terraform configurations.
+
+The HCP Packer validation run task checks the image artifacts within a Terraform configuration. If the configuration references images marked as unusable (revoked), the run task fails and users receive an error message containing the number of revoked artifacts and whether HCP Packer has metadata for newer versions. For HCP Packer Plus registries, run tasks also help you identify hardcoded and untracked images that may not meet security and compliance requirements.
```
Comment (laurapacilio) [inline content/cloud-docs/integrations/run-tasks/index.mdx:85]:
```suggestion
The HCP Packer validation run task checks the image artifacts within a Terraform configuration. If the configuration references images marked as unusable (revoked), the run task fails and provides an error message containing the number of revoked artifacts and whether HCP Packer has metadata for newer versions. For HCP Packer Plus registries, run tasks also help you identify hardcoded and untracked images that may not meet security and compliance requirements.
```

**Pull request merged**